sql/driver/driver_test.go

```diff
@@ -223,7 +223,7 @@ func TestInsecure(t *testing.T) {
 	defer func() {
 		_ = db.Close()
 	}()
-	if _, err := db.Exec("CREATE DATABASE t"); err != nil {
+	if _, err := db.Exec(`SELECT 1`); err != nil {
```

Comment:
for simplicity. we're just testing that we can talk to the server in insecure mode. cc @MBerhault